my_app.py

```diff
@@ -0,0 +1,12 @@
+from omegaconf import DictConfig, OmegaConf
```

Comment:
What is this file for? Doesn't seem to be doing anything?

Reply:
Yes, removed.